core/src/main/java/org/jdbi/v3/SQLStatement.java

```diff
@@ -1058,7 +1062,7 @@ public final SelfType bind(int position, Object value)
      */
     public final SelfType bind(String name, Object value)
     {
-        return bind(name, getForeman().waffle(value != null ? value.getClass() : Object.class, value, getContext()));
+        return bind(name, getForeman().waffle(value == null ? Object.class : value.getClass(), value, getContext()));
```

Comment:
PMD has been drilling this one into me for years: with ternary and if/else statements, avoid `!` in the condition. It makes the flow easier to understand.